guide/onboarding/funding-a-wallet.md

```diff
@@ -0,0 +1,91 @@
+
+### Earning bitcoin
+
+Some users may choose to accept bitcoin as a form of payment. Many people already started accepting bitcoin for their work. Some employers might also choose to pay part of their staff's salary in bitcoin. This is one way to fill a wallet.
+
+> Fun Fact 🏈: Former Seattle Seahawk Russel Okung puts half of his salary in Bitcoin and is now considered one of the highest paid players in the league.
+>
+> <cite>As reported by <a href="https://www.nbcsports.com/northwest/seahawks/former-seattle-seahawk-russell-okung-puts-half-salary-bitcoin-considered-highest">NBC Sports</a></cite>
+
+Pros:
+ - Privacy
+ - No exchange fees
+ - Simplifies onboarding
+
+Cons:
+ - Managing taxes
+ - Difficult to find businesses/individuals willing to pay in bitcoin
+ - Hard to set up recurring payments
+
+### Buying bitcoin
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
+   width = 250
+   height = 250
+   alt-text = "Showing recovery phrase one word at a time"
+   caption = "Buy bitcoin screen from the from [Wallet UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1228%3A28550)"
+   layout = "float-left-desktop -background -shadow"
+%}
+
+As noted, most exchanges offer the option to buy bitcoin using fiat currencies. Once purchased, users can transfer those funds to a separate, external wallet. This might confuse new users who don't understand the difference between exchange wallets and personal wallets.
+
+Bitcoin ATMs are another great option for new users to exchange cash for bitcoin in person. These machines also let users trade their bitcoin for cash, which can be useful if they are somewhere new or foreign.
+
+Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like localbitcoins.
+
+</div>
+
+{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+
+Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+
+{% include /tip/close.html %}
+
+Pros:
+ - An easy and popular way to acquire bitcoin
+
+Cons:
+ - Managing taxes
+ - Difficult to find businesses/individuals willing to pay in bitcoin
+ - Hard to set up recurring paymets
```

Comment:
```suggestion
 - Hard to set up recurring payments
```

spelling. Also, most exchanges offer auto DCA these days so is this really a con?

Reply:
Hmm maybe not, in the UK at least we only have one Bitcoin-only exchange offering reoccurring payments that I know of.

Reply:
Maybe I change to "limited options"